gender_analysis/corpus.py

```diff
@@ -184,7 +185,7 @@ def _load_documents(self):
         documents = []
 
         try:
-            csv_file = self.load_file(self.csv_path)
+            csv_file = load_csv_to_dict(self.csv_path)
```

Comment:
I'd change this variable name as csv_file isn't actually a file, it's a list of strings. (csv_lines or similar)